DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
aha, I see where is the problem:
this line 
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py#L137

should be process.l1tCsctf.gangedME1a = cms.untracked.bool(False)


I will go to make all the changes.